### ISSUE TITLE
[INFRA-1729] Make trusted.ci publicly available from GitHub ip addresses to support webhooks

### ISSUE THREAD

**Issue by jenkins-infra-bot** [jenkins-infra/helpdesk#1446]
<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=danielbeck" class="user-hover" rel="danielbeck">Daniel Beck</a> proposed this PR to make website refresh faster when PR's are merged: <a href="https://github.com/jenkins-infra/jenkins.io/pull/1678" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/jenkins-infra/jenkins.io/pull/1678</a> .  <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=oleg_nenashev" class="user-hover" rel="oleg_nenashev">Oleg Nenashev</a> pointed out the github webhooks would be better than polling every 2 minutes. </p>

<p>If it would be better for this to wait on <a href="https://issues.jenkins-ci.org/browse/INFRA-1352" class="external-link" target="_blank" rel="nofollow noopener">https://issues.jenkins-ci.org/browse/INFRA-1352</a> I have no objection.  I mostly wanted it noted this needed doing. </p>

<p> </p>

<p>The address ranges for github hooks are: </p>

<p>192.30.252.0/22</p>

<p>185.199.108.0/22</p>

<p>140.82.112.0/20</p>

<p>  </p>

<p>References:</p>

<p><a href="https://help.github.com/articles/about-github-s-ip-addresses/" class="external-link" target="_blank" rel="nofollow noopener">https://help.github.com/articles/about-github-s-ip-addresses/</a></p>

<p><a href="https://api.github.com/meta" class="external-link" target="_blank" rel="nofollow noopener">https://api.github.com/meta</a> </p>

---
<details><summary><i>Originally reported by <a title="Liam Newman" href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=bitwiseman">bitwiseman</a>, imported from: <a href="https://issues.jenkins.io/browse/INFRA-1729" target="_blank">Make trusted.ci publicly available from GitHub ip addresses to support webhooks</a></i></summary>
<i><ul>
<li><b>status</b>: Open
<li><b>priority</b>: Minor
<li><b>resolution</b>: Unresolved
<li><b>imported</b>: 2022/01/10
</ul></i>
</details>

**Comment by dduportal** [comment]
@daniel-beck do you think webhook are still needed on trusted.ci ? and on cert.ci ? 
Or shall we close this issue as "Not Planned" ?

**Comment by dduportal** [comment]
> Assuming this is far from trivial, given it's age, IMO this can be closed.

Agreed. We can revisit later, but opening this publicly can be risky while the gain is not that high.
